src/app/App.tsx

```diff
@@ -21,7 +23,13 @@ const router = createBrowserRouter(
         <Route
             path='/'
             element={
-                <Suspense fallback={<div>Please wait while we load the app...</div>}>
+                <Suspense
+                    fallback={
+                        <div className='app-root'>
+                            <Loader />
+                        </div>
```

Comment:
```suggestion
             <div className='app-root'>
              <Loader />
            <div>{localize('Please wait while we connect to the server...')}</div>
        </div>
```

@mayuran-deriv let's use this 🙏